draft-ietf-snac-simple.xml

```diff
@@ -859,6 +859,66 @@
     </section>
   </middle>
   <back>
+    <section>
+      <name>Analysis of deployment scenarios in which a stub router could cause problems</name>
+      <section>
+	<name>Unmanaged home network</name>
+	<t>
+	  In this scenario, a non-expert home user connects stub router to own unmanaged home network. This is the key intended use case for stub networks.
+	  This document describes how to implement a stub router such that it operates correctly in this situation, whether the ISP is providing IPv6
+	  reachability to the Internet or not.
+	</t>
+      </section>
+      <section>
+	<name>Use on properly managed network</name>
+	<t>
+	  In this scenario, a non-expert user attaches a stub router to an infrastructure network that's managed. This network has correctly deployed
+	  RA Guard and/or port-based access control. As a result, the stub router won't succeed in advertising a prefix on the managed network.
+	</t>
+	<t>
+	  In the managed network case, it's possible that the network operator is willing to permit stub routers to be attached to the network by
+	  users. In this case, they might either not deploy RA guard, or they might deploy working DHCPv6 prefix delegation. This could be
+	  PD-per-host (where hoses are encouraged to use prefix delegation) or just ordinary prefix delegation (where hosts are given prefix
+	  delegation if they ask for it, but not encouraged to ask for it).
+	</t>
+	<t>
+	  In such a situation, if mDNS works on the infrastructure link, the stub router will function correctly, because the delegated prefix
+	  will be correctly routed.
+	</t>
+	<t>
+	  It's worth noting that IPv4 devices that act similarly to stub routers, using NAT64, already exist and may indeed use the stub network
+	  functionality to support internal connections that aren't even apparent to the user. So it's not implausible that it might be desirable
```

Comment:
Sentence is very complex "not implausible that it might be desirable to enable ... "